semeio/jobs/scripts/csv_export2.py

```diff
@@ -41,6 +32,6 @@ def _get_args_parser():
     parser = argparse.ArgumentParser()
     parser.add_argument("runpathfile", type=str)
     parser.add_argument("outputfile", type=str)
-    parser.add_argument("datesetting", type=str)
-    parser.add_argument("vectors", nargs="+")
+    parser.add_argument("--vectors", nargs="+")
+    parser.add_argument("--datesetting", type=str, default="monthly")
```

Comment:
We should perhaps change these to fmu-ensemble's object names, so `--column_keys` and `--time_index` as option names (this is in turn derived from libecl..pandas_frame()). This makes it similar to the syntax of `ecl2csv summary` https://equinor.github.io/ecl2df/scripts.html#summary